Entities/Cliente.cs

```diff
@@ -84,10 +101,52 @@ public void ValidarCep()
             c = this.Cep.Substring(i, 1);
             if (c == "-" && i == 5) digito = 5;
             else y += c;
-
          }
          if (int.TryParse(y, out x) && digito == 5 && this.Cep.Length == 9) { }
          else throw new Exception("CEP não está no formato válido \"00000-000\"");
       }
+
+      public static List<Cliente> LerBancoClinete()
+      {
+         List<Cliente> clientes = new List<Cliente>();
+         ca.RunQuery("SELECT idCliente, Nome, Endereco, Cidade FROM Clientes ORDER BY Nome", (OleDbDataReader dr) =>
+            {
+               while (dr.Read())
+               {
+                  Cliente cliente = new Cliente {
+                     IdCliente = Convert.ToInt32(dr["idCliente"]),
+                     Nome = dr["Nome"].ToString(),
+                     Endereco = dr["Endereco"].ToString(),
+                     Cidade = dr["Cidade"].ToString(),
+                  };
+                     clientes.Add(cliente);
+               }
+            }
+         );
+         
+         return clientes;
+      }
+
+      public static Cliente GetCliente(int idcliente)
+      {
+         Cliente cliente = new Cliente();
+         ca.RunQuery($@"SELECT c.idCliente, c.Nome, c.Endereco, c.Cidade, c.Cep, e.idEstado, e.Estado, p.idEstadoCivil, p.EstadoCivil
+                      FROM((Clientes c INNER JOIN
+                         Estados e ON e.idEstado = c.idEstado) INNER JOIN
+                         EstadoCivil p ON p.idEstadoCivil = c.idEstadoCivil) WHERE(c.idCliente = {idcliente})", (OleDbDataReader dr) =>
```

Comment:
Renomear para manter o padrão lower camelCase

```suggestion
      public static Cliente GetCliente(int idCliente)
      {
         Cliente cliente = new Cliente();
         ca.RunQuery($@"SELECT c.idCliente, c.Nome, c.Endereco, c.Cidade, c.Cep, e.idEstado, e.Estado, p.idEstadoCivil, p.EstadoCivil
                      FROM((Clientes c INNER JOIN
                         Estados e ON e.idEstado = c.idEstado) INNER JOIN
                         EstadoCivil p ON p.idEstadoCivil = c.idEstadoCivil) WHERE(c.idCliente = {idCliente})", (OleDbDataReader dr) =>
```